internal/gps/vcs_repo.go

```diff
@@ -37,30 +38,26 @@ func newCtxRepo(s vcs.Type, ustr, path string) (r ctxRepo, err error) {
 		r, err = getVCSRepo(s, ustr, path)
 	}
 
-	return
```

Comment:
Since you replaced the naked return, there is no need for named return variables.